golang/cosmos/daemon/cmd/root.go

```diff
@@ -133,6 +135,15 @@ func initRootCmd(sender Sender, rootCmd *cobra.Command, encodingConfig params.En
 	}
 	server.AddCommands(rootCmd, gaia.DefaultNodeHome, ac.newApp, ac.appExport, addModuleInitFlags)
 
+	for _, command := range rootCmd.Commands() {
+		if command.Name() != "export" {
+			continue
+		}
+
+		extendCosmosExportCommand(command)
+		break
+	}
```

Comment:
```suggestion
	// Only do the following modifications if we actually have a controller.
	// Otherwise, there will be an exec of the VM program, and it will
	// handle all of the export logic.
	if sender != nil {
		for _, command := range rootCmd.Commands() {
			if command.Name() != "export" {
				continue
			}

			extendCosmosExportCommand(command)
			break
		}
	}
```